tests/cbmc/stubs/write.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cbmc_proof/nondet.h>
+#include <unistd.h>
+
+static ssize_t round = MAX_BLOB_SIZE;
+
+ssize_t write(int fildes, const void *buf, size_t nbyte) {
+    round--;
+    return (round > 0) ? nondet_ssize_t() : round;
```

Comment:
See comment for `read`

Reply:
https://github.com/awslabs/s2n/pull/2071#discussion_r457732766